http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppComponent.scala

```diff
@@ -204,7 +207,8 @@ object appQuery extends TableQuery(new AppTable(_)) {
         diskOpt.map(_.id),
         if (saveApp.app.customEnvironmentVariables.isEmpty) None else Some(saveApp.app.customEnvironmentVariables),
         saveApp.app.descriptorPath,
-        if (saveApp.app.extraArgs.isEmpty) None else Some(saveApp.app.extraArgs)
+        if (saveApp.app.extraArgs.isEmpty) None else Some(saveApp.app.extraArgs),
+        if (saveApp.app.sourceWorkspaceId.isEmpty) None else saveApp.app.sourceWorkspaceId
```

Comment:
I think this is redundant. If `saveApp.app.sourceWorkspaceId.isEmpty` is true, then `saveApp.app.sourceWorkspaceId` will be a `None`. So you can just use:
```suggestion
        saveApp.app.sourceWorkspaceId
```
similar to how `saveApp.app.descriptorPath` is specified at line 209.